src/pages/TodoList.js

```diff
@@ -0,0 +1,210 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import ListContainer from '../components/ListContainer';
+import { TODOKEY } from '../constants/TODOKEY';
+
+const TodoList = () => {
+  const [inputText, setInputText] = useState('');
+  const [todoList, setTodoList] = useState(
+    JSON.parse(localStorage.getItem(TODOKEY)) ?? []
+  );
+  const [numOfTodo, setNumOfTodo] = useState(0);
+  const [numOfDone, setNumOfDone] = useState(0);
+  const [selectedTodo, setSelectedTodo] = useState({});
+  const [isVisibleModal, setIsVisibleModal] = useState(false);
+
+  useEffect(() => {
+    setNumOfTodo(todoList.filter((todo) => !todo.isDone).length);
+    setNumOfDone(todoList.filter((todo) => todo.isDone).length);
+  }, [todoList]);
+
+  const handleInputChange = (e) => {
+    setInputText(e.target.value);
+  };
+
+  const handleInputKeyDown = (e) => {
+    if (e.key === 'Enter') {
+      submitInput();
+    }
+  };
+
+  const submitInput = () => {
+    if (inputText.trim()) {
+      const todoObj = {
+        text: inputText, // string
+        time: String(Date.now()), // string,
+        isDone: false, // boolean
+      };
+      setTodoList([...todoList, todoObj]);
+      setInputText('');
+
+      localStorage.setItem(TODOKEY, JSON.stringify([...todoList, todoObj]));
+    }
+  };
```

Comment:
아래 comment와 연결되는 수정사항입니다! (웹에서 수정해서 제대로 돌아갈지는 모르겠어요ㅠ)
```suggestion
  const handleSubmit = () => {
    if (inputText.trim()) {
      const todoObj = {
        text: inputText, // string
        time: String(Date.now()), // string,
        isDone: false, // boolean
      };
      setTodoList([...todoList, todoObj]);
      setInputText('');

      localStorage.setItem(TODOKEY, JSON.stringify([...todoList, todoObj]));
    }
  };
```
  ****